goerli/357558.json

```diff
@@ -0,0 +1 @@
+[{"message":{"validator_index":"357558","from_bls_pubkey":"0x90e6b8bb3d9fc5a69bd800c8ce7545ac4611274b89f85c9fb2b55c6ad7c685076991e7aeba0f2ad2f974f815ead97368","to_execution_address":"0x5385eefdae761edc311f08474a6ca0bf2eb8f2a7"},"signature":"0xa93f938fcc5ce676a006687b191be74082c8c48fe29aadc6f87819a254e52046b5153a41f297b48202f5cf43edc96a2c0d9a22e2fb3a94a0aa10e64474fc1e1f433a25108bb8c0a7cdda069a2f2bc4dd5dbcc3b2c83397da7c41b7b393c80087"}]
```

Comment:
Invalid signature for goerli on validator 357558. Please try again? If you need further assistance, ask in the Discord group.